pennylane/visualize/visualize.py

```diff
@@ -0,0 +1,351 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
```

Comment:
I know this is a WIP, bit it's best to handle these codefactor issues from the beginning!

Reply:
Absolutely, I'll fix this right away